sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobContainerClientBuilder.java

```diff
@@ -104,8 +138,8 @@ public BlobContainerClientBuilder endpoint(String endpoint) {
             this.containerName = parts.getBlobContainerName();
 
             String sasToken = parts.getSasQueryParameters().encode();
-            if (ImplUtils.isNullOrEmpty(sasToken)) {
-                super.sasToken(sasToken);
+            if (!ImplUtils.isNullOrEmpty(sasToken)) {
```

Comment:
Was this a bug in the previously released preview?

Reply:
Yes, this appears to have been release with the incorrect boolean check.